pyvista/core/pointset.py

```diff
@@ -606,9 +604,17 @@ def _from_arrays(self, offset, cells, cell_type, points, deep=True):
 
     @property
     def cells(self):
-        """Return a pointer to the cells as a numpy object."""
+        """Legacy method: Return a pointer to the cells as a numpy object."""
         return vtk_to_numpy(self.GetCells().GetData())
 
+    @property
+    def cell_connectivity(self):
+        """Return a the vtk cell connectivity as a numpy array."""
+        carr = self.GetCells()
+        if hasattr(carr, 'GetConnectivityArray'):  # available >= VTK9
+            return vtk_to_numpy(carr.GetConnectivityArray())
+        return None
```

Comment:
Should we alert users that this isn't available and that they need VTK9? Perhaps with an `AttributeError`?

I can see trying to use this property in a workflow and it returning `None` and the code after it would break. This behavior would confuse users and it wouldn't be obvious that they need VTK9

Reply:
Valid point.  Informative errors are better than `None`.  I'm not letting them know they should use the legacy `cells` method when using `vtk<9.0.0`.